examples/building_and_installing_viame/README.rst

```diff
@@ -214,8 +208,8 @@ Building on Windows
 
 Building on windows can be very similar to Linux if using a shell like cygwin
 (``https://www.cygwin.com/``), though if not you may want to go grab the GUI
-ersions of CMake (``https://cmake.org/``) and TortoiseGit (``https://tortoisegit.org/``).
-Currently Visual Studio 2015 and some versions of 2017 are supported.
+versions of CMake (``https://cmake.org/``) and TortoiseGit (``https://tortoisegit.org/``).
+Currently Visual Studio 2015 and 2017 are supported.
```

Comment:
I've built it on a recent VS 2017

Reply:
I would update the main page (default README.rst) to say that too then